src/components/common/AppMenu.vue

```diff
@@ -258,6 +258,12 @@ export default {
   font-weight: 500;
 }
 
+@media screen and (max-width: 667px) {
```

Comment:
is there a reason you don't just apply `hide-xs` to line 81?

Reply:
Clumsiness? :laughing: